setup.py

```diff
@@ -0,0 +1,73 @@
+import os
+
+from setuptools import setup
+
+from neptune_sacred._version import get_versions
+
+
+def main():
+    with open('README.md') as readme_file:
+        readme = readme_file.read()
+
+    extras = {}
+
+    all_deps = []
+    for group_name in extras:
+        all_deps += extras[group_name]
+    extras['all'] = all_deps
+
+    base_libs = ['neptune-client>=0.9.7', 'sacred', 'flatten-dict']
```

Comment:
Do we actually need a dependency on `flatten-dict`? Flattening a dict sounds easy to implement in a short method.

Reply:
Yes we can, but why not to use ready library with ~100 stars?

Reply:
This is my thinking: if each library depends on one more library than strictly necessary, then a user ends up with polluted environment, version conflicts etc.

In this case, we can simply have a method

```python
def flatten_dict(dictionary, delimiter='/'):
    result = {}
    segments = []
    stack = [dictionary]
    while stack:
        elem = stack.pop()
        for key, value in elem.items():
            if isinstance(value, dict):
                segments.append(key)
                stack.append(value)
            else:
                segments.append(key)
                result[delimiter.join(segments)] = value
                segments.pop()
    return result
```

instead of another dependency

E.g. try

```
flatten_dict({
    'a': {
        'b': 1,
        'c': {
            'd': 2,
            'e': 3
        }
    },
    'f': 4
})
```

to get 

```
{'a/f': 4, 'a/b': 1, 'a/c/d': 2, 'a/c/e': 3}
```

Reply:
I had the same thoughts, but when I've figured out that it's not trivial two lines of code I've decided to use something ready.

Your code returns `'a/f': 4` instead of `'f': 4`, that's why I didn't implemented it on my own :p